public/javascript/index.js

```diff
@@ -1,23 +1,101 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
+const charactersAPI = new APIHandler
```

Comment:
Me sorprende MUCHISIMO que esto haya funcionado.

Ese new ApiHandler es una clase, un constructor o **función** constructora.

Cómo tal no la estás invocando.

```suggestion
const charactersAPI = new APIHandler()
```